.github/workflows/nightly-test-integrations-1.18.x.yml

```diff
@@ -109,7 +109,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        envoy-version: ["1.24.12", "1.25.11", "1.26.8", "1.27.5"]
+        envoy-version: ["1.24.12", "1.25.11", "1.26.8", "1.27.6"]
```

Comment:
Wondering if this list should have included `1.28.5`

Reply:
Good observation, since we have a task for the next release to bump Envoy to all the latest and greatest stable versions we decided to update the currently existing ones to fix the CVEs so we have less friction for the current release.